TheMovieDb/Controllers/ListViewController.swift

```diff
@@ -123,13 +148,16 @@ class ListViewController: UICollectionViewController {
         self.activityIndicator.isHidden = true
     }
     
+}
+
+extension ListViewController: UICollectionViewDataSource, UICollectionViewDelegate {
     // MARK: - CollectionView DataSource
-    override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         return movieList?.results?.count ?? 0
     }
     
-    override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: MovieCell.cellIdentifier, for: indexPath) as? MovieCell else {
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: MovieCellController.cellIdentifier, for: indexPath) as? MovieCellController else {
             preconditionFailure("Failed to load collection view cell")
```

Comment:
Nice, sometimes it is preferible to crash to have the app let us know that something is not configured appropiately, rather than having a silent fail. 👍